serde/src/java/org/apache/hadoop/hive/serde2/objectinspector/ObjectInspector.java

```diff
@@ -41,15 +42,6 @@
 @InterfaceAudience.Public
 @InterfaceStability.Stable
 public interface ObjectInspector extends Cloneable {
-
-  /**
-   * Category.
-   *
-   */
```

Comment:
Won't this change break anyone with their own ObjectInspector?  For example, a quick perusal of the Parquet code looks like it uses this quite frequently.

Reply:
Yeah, I just check the Parquet code and I think you are right. I didn't know that Parquet (and now possibly other codebases outside hive) would be using this class. In that case, I think we will have to copy (or map it to a new Category enum in the standalone-metastore). It might be little hacky but I can't think of a better way. I will give it shot and update the patch today. Thanks for catching this!

Reply:
Hi Alan, I updated the PR with the latest changes. Basically instead of moving Category enum out of ObjectInspector, I moved ObjectInspector to storage-api which can be accessed from standalone-metastore. This way the change becomes even smaller and also maintain backwards compatibility.